subprojects/platform-play/src/main/java/org/gradle/play/tasks/TwirlCompile.java

```diff
@@ -76,6 +76,7 @@
     private PlayPlatform platform;
     private List<TwirlTemplateFormat> userTemplateFormats = Lists.newArrayList();
     private List<String> additionalImports = Lists.newArrayList();
+    private List<String> constructorAnnotations = Lists.newArrayList();
```

Comment:
Like the source set, this should be a final `ListProperty` too and created via `project.getObjects().listProperty(String.class)`